app/src/main/java/com/github/wnder/AlertBuilder.java

```diff
@@ -0,0 +1,24 @@
+package com.github.wnder;
+
+
+import android.content.Context;
+import android.content.DialogInterface;
+
+import androidx.appcompat.app.AlertDialog;
+
+public final class AlertBuilder {
+    private AlertBuilder(){
+        //Non instanciable class
+    }
+
+    public static AlertDialog createAlert(String title, String body, Context ctx){
```

Comment:
As the alert is always the same (for now at least) could we just create a "noConnectionAlert(Context ctx)"? (That would reuse this method)

Reply:
The body changes so it is not possible to do that

Reply:
I though it was the same everywhere mb.